diesel/src/types/impls/tuples.rs

```diff
@@ -250,6 +250,18 @@ macro_rules! tuple_impls {
                     ($($T,)+ next)
                 }
             }
+
+            #[cfg(feature = "postgres")]
+            impl<$($T,)+ ST> $crate::pg::expression::array::IntoSingleTypeExpressionList<ST> for ($($T,)+) where
```

Comment:
There's no need to use fully qualified paths vs just `use`ing the trait here. This macro doesn't get called from user code.

Reply:
:+1: